profanity_word_phrase_filter/README.md

```diff
@@ -1,19 +1,19 @@
 ## Custom profanity filter
 
-Most online platforms contain inappropriate language. Parent's want to be certain
+Most online platforms contain inappropriate language. Parents want to be certain
 that Story squad (aka scribble stadium) is offering a safe environment for their
-kiddos. In compliance to the children's Online Protection rule (COPPA)
+kiddos in compliance with the Children's Online Privacy Protection Rule (COPPA).
 https://www.ftc.gov/enforcement/rules/rulemaking-regulatory-reform-proceedings/childrens-online-privacy-protection-rule
 
-It's important for the submitted stories to be flagged as inappropriate if they 
-contain any profanity. The stories will still be viewed by a human eye to make
-sure that the flagging is warranted. 
-The previous cohort worked on using the available libraries like profanity-filter
-and profanity_Check but they were not sufficient in flagging all words and newer
+It is important for submitted stories to be flagged as inappropriate if they 
+contain any profanity. The stories will still be reviewed by a live person to make
+sure the flagging is warranted. 
+The previous cohort worked on using available libraries like profanity-filter
+and profanity_Check but these could not sufficiently flag all profane words, newer
 phrases and Scuntorpe like words.
 
-Using a list of profanity phrases in (bad_phrases.csv) and profanity words in (bad_single.csv).
+Using a list of profanity phrases in (bad_phrases.csv) and profanity words in (bad_single.csv);
 bad_phrases.csv has a total of 215 phrases/expressions and bad_single.csv has a total of 1395 words.
-These words can updated according to reflect what is currently flagged as inappropriate.
-profanity_word_phrase_filters.py uses both lists to search if an uploaded document
+These words are updated accordingly to reflect what is currently flagged as inappropriate.
+Profanity_word_phrase_filters.py uses both lists to search if an uploaded document
```

Comment:
The file name is profanity_word_phrase_filters.py starting with a lowercase p. It may be less ambiguous by not starting a sentence with the filename. Maybe try "Both lists are used to check for profanity when running profanity_word_phrase_filters.py." Semantics. Great job updating a difficult to read README.md file.

Reply:
Made changes to this as well. Sounds much better.